pages/resources/qbx_core/modules/lib/client.mdx

```diff
@@ -35,14 +39,18 @@ qbx.drawText3d(params)
   - text: `string`
   - coords: `vector3`
     - In-world coordinates.
-  - scale?: `integer`
-    - Default: `0.35`
+  - scale?: `vector2`
+    - Default: `vec2(0.35, 0.35)`
```

Comment:
Can you also reflect that scale as an integer still works? I think we can support both.

Reply:
Will do.

Reply:
Would you like me to give examples? Or is it enough to change it to 
```lua
  - scale?: `integer` | `vector2`
    - Default: `vec2(0.35, 0.35)`

Reply:
You don't need to give examples, up to you.

Reply:
@Manason the newest commit should fix it. Can you check it to see if it's to your liking?